### PR TITLE
Update: "tool data section" and "tech blog" feed to show updated information

### PR DESCRIPTION
**Pull request by Claurt07**
Update Tools and Data Section + Tech Blog

Changes: 

**Tools section**
- Changed titles and removed tools that are no longer supported
- Chagned picture from one section

**Tech blog**
- Changed the "tech blog" and the "tech blog archive" to be fed from both the news section and the blog folder
  - To further reduce the scope, I added the “tech” as a filter 
  - Tagged many relevant articles from the news section

Screenshots of the change: 
image

Note: approve this one after the PR for program pages

### CODE REVIEW

**Comment by Claurt07** [comment]
@dakotabenjamin @kshitijrajsharma Sorry to send another one so soon, but Leen asked me this asap. 
Note: approve this one after the PR for program pages.

**Comment by kshitijrajsharma** [comment]
@Claurt07  You can use github conventions like following to mention dependencies

- Blocked by #845

File: _programs/mapping-for-climate-ready-cities.markdown [hunk 1]
```diff
@@ -1,75 +1,25 @@
 ---
-title: Mapping for Climate-Ready Cities
+title: 'Mapping for Climate-Ready Cities'
 date: 2018-05-01 05:01:00 Z
-permalink: "/programs/mapping-for-climate-ready-cities"
+permalink: /programs/mapping-for-climate-ready-cities
```
Comment (kshitijrajsharma) [inline _programs/mapping-for-climate-ready-cities.markdown:4]: Intended ?

Comment (dakotabenjamin) [reply _programs/mapping-for-climate-ready-cities.markdown:4]: quotes are not necessary unless there are special characters

**Pull request merged**